### ISSUE TITLE
ArduinoJson::JsonArray has no member named 'clear'

### ISSUE THREAD

**Issue by bhcuong2008** [bblanchon/ArduinoJson#1597]
Hi,

I create a JsonArray as below, then got error `'ArduinoJson::JsonArray {aka class ArduinoJson6180_D1::ArrayRef}' has no member named 'clear'`

My target platform: ESP32 Arduino 1.06, IDF 3.3.5
ArduinoJson 6.18

```
StaticJsonDocument<512> test;
JsonArray ja = test.as<JsonArray>();
ja.clear();
```
I dont know why because the document says it has method clear.

Thanks.

**Comment by bblanchon** [comment]
Hi @bhcuong2008,

Thank you very much for reporting this bug 👍 
I fixed it in the main branch.

Best regards,
Benoit

**Comment by bhcuong2008** [comment]
Thank u very much for your fast fix.

**Comment by bblanchon** [comment]
You're welcome, @bhcuong2008!
Thank you for using ArduinoJson.
Don't forget to [cast a star](https://github.com/bblanchon/ArduinoJson/stargazers) to support the project :wink:

**Comment by bblanchon** [comment]
The fix is included in [ArduinoJson 6.18.1](https://github.com/bblanchon/ArduinoJson/releases/tag/v6.18.1)
